core/src/avm1/activation.rs

```diff
@@ -870,14 +870,16 @@ impl<'a, 'gc> Activation<'a, 'gc> {
         let swf_version = self.swf_version();
         let func_data = parent_data.to_unbounded_subslice(action.actions);
         let constant_pool = self.constant_pool();
+        let bc = self.base_clip.object().coerce_to_object(self);
         let func = Avm1Function::from_swf_function(
             self.gc(),
             swf_version,
             func_data,
             action,
             self.scope(),
             constant_pool,
-            self.base_clip(),
+            // `base_clip` should always be a living `MovieClip` so this can't fail
+            MovieClipReference::try_from_stage_object(self, bc).unwrap(),
```

Comment:
I'm open for an alternative to the `unwrap`, but I've manged to convince myself it's safe